references/cli/status.go

```diff
@@ -182,6 +198,7 @@ func NewAppStatusCommand(c common.Args, order string, ioStreams cmdutil.IOStream
 	cmd.Flags().BoolVarP(&detail, "detail", "d", false, "display more details in the application like input/output data in context. Note that if you want to show the realtime details of application resources, please use it with --tree")
 	cmd.Flags().StringP("detail-format", "", "inline", "the format for displaying details, must be used with --detail. Can be one of inline, wide, list, table, raw.")
 	cmd.Flags().StringVarP(&outputFormat, "output", "o", "", "raw Application output format. One of: (json, yaml, jsonpath)")
+	cmd.Flags().BoolP("metrics", "m", false, "show resource quota and consumption metrics of the application")
```

Comment:
since App may expose customized metrics except CPU/MEM/storage, can this command show them?

Reply:
Now I think the answer is not.

Reply:
Like some prometheus metrics exported in `http:<endpoint>:8084/metrics`? How do you think @StevenLeiZhang